salt/pillar/vault.py

```diff
@@ -25,6 +25,13 @@
 Each key needs to have all the key-value pairs with the names you
 require. Avoid naming every key 'password' as you they will collide:
 
+If you want to nest results under a nesting_key name use the following format:
+
+    ext_pillar:
+      - vault: 
+          conf: path=secret/salt
+          nesting_key: vault_key_name
```

Comment:
can we change `nesting_key` to `pillar`  and then document that the values pulled out of the ext_pillar will be placed under whatever `pillar` is specified as?